tst/schedulers/multiobjective/test_utils.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import numpy as np
+
+from syne_tune.optimizer.schedulers.multiobjective.utils import (
+    hypervolume,
+    hypervolume_cumulative,
+)
+
+
+def test_hypervolume():
+    ref_point = np.array([2, 2])
+    points = np.array([[1, 0], [0.5, 0.5], [0, 1], [1.5, 0.75]])
+    hv = hypervolume(results_array=points, reference_point=ref_point)
+    assert hv == 3.25
```

Comment:
Just for the sake of verifying that this new function works correctly, how do we know that the magic number of 3.25 is correct here? Can I compute this example in my head, or is there an even simpler example that could also be used to verify the computation?

Reply:
I suppose you can compute this in your head but using a piece of paper is simpler :) I think this example strikes the right balance between simplicity and not being trivial but I can add a simpler case.